Pipfile

```diff
@@ -26,6 +26,11 @@ djangorestframework-bulk = "~=0.2.1"
 psycopg2-binary = "~=2.8"
 django-simple-bulma = ">=1.1.6,<2.0"
 django-crispy-bulma = ">=0.1.2,<2.0"
+requests = "*"
```

Comment:
This should be fixed so we stay on the same major version

Reply:
reasonable enough. locking to `requests = "~=2.21"`